crates/uv/src/commands/tool/install.rs

```diff
@@ -276,19 +276,27 @@ pub(crate) async fn install(
         installed_tools
             .get_environment(&from.name, &cache)?
             .filter(|environment| {
-                python_request.as_ref().map_or(true, |python_request| {
-                    if python_request.satisfied(environment.interpreter(), &cache) {
-                        debug!("Found existing environment for `{from}`", from = from.name.cyan());
-                        true
-                    } else {
-                        let _ = writeln!(
-                            printer.stderr(),
-                            "Existing environment for `{from}` does not satisfy the requested Python interpreter",
-                            from = from.name.cyan(),
-                        );
-                        false
-                    }
-                })
+                // NOTE(lucab): this compares `base_prefix` paths as a proxy for
```

Comment:
As a style nit, we don't usually attach names to notes. Something like "N.B." is more common.

cc @charliermarsh not sure if you have a stance here.

Reply:
I can drop that, but for reference this form is already in use in this codebase: https://github.com/search?q=repo%3Aastral-sh%2Fuv%20NOTE(&type=code

I don't remember exactly where I picked up this habit, I personally like it. It spares a bunch of recursive git-blaming to find out who wrote a specific note/todo and provides a breadcrumb in case of future doubts.

Reply:
Ah fair point — I don't mind keeping it. Thanks for correcting me.